<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcquisitionWidgetHorizontalLayout</class>
 <widget class="QWidget" name="AcquisitionWidgetHorizontalLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Acquisition widget</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QCheckBox" name="osc_start_cbox">
             <property name="text">
              <string>Oscillation start:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="first_iamge_label">
             <property name="text">
              <string>First image:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="transmission_label">
             <property name="text">
              <string>Transmission (%):</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="exp_time_label">
             <property name="text">
              <string>Exposure time:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLineEdit" name="osc_start_ledit">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>32767</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="first_image_ledit">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>32767</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="transmission_ledit">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>32767</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="exp_time_ledit">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>32767</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="col_one_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLabel" name="osc_range_label">
             <property name="text">
              <string>Oscillation range:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="num_images_label">
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Number of images:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="resolution_label">
             <property name="text">
              <string>Resolution (Ã…):</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLineEdit" name="osc_range_ledit">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>32767</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="num_images_ledit">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>32767</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="resolution_ledit">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>32767</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="col_two_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLabel" name="overlap_label">
             <property name="text">
              <string>Oscillation overlap:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="num_passes_label">
             <property name="text">
              <string>Number of passes:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="energy_label">
             <property name="text">
              <string>Energy (KeV):</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="col_thee_spacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>30</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLineEdit" name="overlap_ledit">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>32767</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="num_passes_ledit">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>32767</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="energy_ledit">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>32767</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QCheckBox" name="mad_cbox">
               <property name="text">
                <string>MAD</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="energies_combo"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="col_three_vspacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer18">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="row_one_two_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>37</number>
     </property>
     <item>
      <widget class="QCheckBox" name="shutterless_cbx">
       <property name="text">
        <string>Shutterless</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="row_three_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QCheckBox" name="inverse_beam_cbx">
       <property name="text">
        <string>Inverse beam</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="subwedge_size_label">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Subwedge size:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="subwedge_size_ledit">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>32767</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="subwedge_fill_spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>322</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vspacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
